pyproject.toml

```
@@ -56,6 +56,11 @@ ESEF = [
 ObjectMaker = [
     'graphviz==0.*',
 ]
+Sphinx = [
+    "myst-parser==2.*",
+    "sphinx==6.*",
+    "furo==2023.*",
```

furo is like the regex library above, the version doesn't provide any compatibility meaning. Best to drop the version requirement.
```suggestion
    "furo",
```